src/mcts/search.cc

```diff
@@ -381,35 +392,46 @@ std::pair<Move, Move> Search::GetBestMoveInternal() const
   return {best_node.GetMove(played_history_.IsBlackToMove()), ponder_move};
 }
 
-// Returns a child with most visits.
 EdgeAndNode Search::GetBestChildNoTemperature(Node* parent) const {
-  EdgeAndNode best_edge;
+  return GetAscChildrenNoTemperature(parent).back();
+}
+
+EdgeAndNode Search::GetBestChildWithTemperature(Node* parent,
+                                                   float temperature) const {
+  return GetAscChildrenWithTemperature(parent, temperature).back();
```

Comment:
It's simpler code to do this, but is it worth the cpu time of fully sorting the list instead of just finding the single best one every time? This function is called once for every playout, it might be worth the optimization to keep them both separate